src/modules/fetchlike.js

```diff
@@ -0,0 +1,7 @@
+export default async function fetchlike() {
+  return fetch(
+    'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/1RjuQJrVTvT45Mcm7X6n/likes',
+  )
+    .then((res) => res.json())
+    .then((output) => output);
+}
```

Comment:
Thank you for your feedback sir. Sir, according to the guidelines I believe you may have missed out on fourth point.
![image](https://user-images.githubusercontent.com/108437454/197293159-5bfb9742-7cca-4aec-83d5-9a71ee82f1ec.png)

Kindly have a look as it would be very hard to convert promises because currently I am working on a deadline. I'll surely try implementing it later on. I believe it would enhance my coding skills and make everything look better.